deepmd/pd/train/training.py

```
@@ -18,13 +19,16 @@
 from paddle.distributed import (
     fleet,
 )
+from paddle.distributed.fleet.utils import hybrid_parallel_util as hpu
```

Ruff (0.12.2)

`paddle.distributed.fleet.utils.hybrid_parallel_util` imported but unused

Remove unused import: `paddle.distributed.fleet.utils.hybrid_parallel_util`

(F401)